care/facility/api/viewsets/facility.py

```diff
@@ -126,6 +127,25 @@ def get_users(self, request, external_id):
         data = UserAssignedSerializer(users, many=True)
         return Response(data.data)
 
+    @action(methods=["POST", "DELETE"], detail=True)
+    def cover_image(self, request, external_id):
+        facility = Facility.objects.filter(external_id=external_id).first()
```

Comment:
use the get_object method of the viewset.

Reply:
got it